src/app/Main/Body/QueryPage/Drawer/Filter/Filter-Container.component.tsx

```diff
@@ -23,8 +24,10 @@ export function FilterContainer(props: {
     ) => void;
     onFilterRemoveAll: () => void;
     onSubmitFiltersToDisplay: (newFiltersToDisplay: string[]) => void;

```

Comment:
This was a left over from the refactoring which is used again now?

Reply:
Yes.